sklearn/datasets/base.py

```diff
@@ -520,3 +521,173 @@ def load_sample_image(image_name):
     if index is None:
         raise AttributeError("Cannot find sample image: %s" % image_name)
     return images.images[index]
+
+def load_kddcup(set=None):
```

Comment:
I think it would be best to name this `load_kddcup99`, as there are plenty of KDDCup datasets.